coalib/settings/Setting.py

```diff
@@ -155,12 +156,13 @@ class Setting(StringConverter):
     def __init__(self,
                  key,
                  value,
-                 origin: str = '',
+                 origin=SourcePosition(''),
```

Comment:
Dont remove type information.  And it must be backwards compatible, which means it still must handle `str`.

> origin: (str, SourcePosition) = ''